tests/unit_tests/experiment_plans/test_grid_detect_then_xray_centre_plan.py

```diff
@@ -30,7 +28,7 @@ def _fake_grid_detection(
     box_size_um: float = 0.0,
 ):
     oav = i03.oav(fake_with_ophyd_sim=True)
-    smargon = fake_smargon()
+    smargon = i03.smargon(fake_with_ophyd_sim=True)
```

Comment:
Should: This isn't equivalent as `fake_smargon` patched the motors too. Can you use the `smargon()` in `conftest` here?